### PR TITLE
Avishay/mgmt 908 - APIs for kubeconfig

### PR DESCRIPTION
**Pull request by avishayt**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -264,6 +264,73 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /clusters/{cluster_id}/downloads/kubeconfig:
```
Comment (tsorya) [inline swagger.yaml:267]: don't u think better to call it kubeconfig-withingress ?
less changes :)

Comment (avishayt) [reply swagger.yaml:267]: No, don't be lazy ;)
Users expect kubeconfig to work with ingress.  They shouldn't need to know about it even.  The strange one is kubeconfig-noingress which will not work for ingress.

File: swagger.yaml [hunk 2]
```diff
@@ -264,6 +264,73 @@ paths:
...
+        - multipart/form-data
+      parameters:
+        - in: path
+          name: cluster_id
+          type: string
+          format: uuid
+          required: true
+        - in: formData
+          name: upfile
+          type: file
```
Comment (tsorya) [inline swagger.yaml:316]: why file? lets just make it string

Comment (tsorya) [reply swagger.yaml:316]: it is relatively small amount of data, not sure it worth to be file, it will never saved as file

**Comment by filanov** [comment]
is that a WIP? where is the implementation?

**Pull request closed without merging**